tests/acceptance/standard/test_resource_groups.py

```diff
@@ -38,9 +40,59 @@ def test__update_resource_group_process_mode(self, project, add_gitlab_ci_config
                 process_mode: newest_first
             """
 
-        time.sleep(5)
         run_gitlabform(update_resource_group_config, project)
+        time.sleep(5)
 
         resource_group = project.resource_groups.get("production")
         assert resource_group.key == "production"
         assert resource_group.process_mode == "newest_first"
+
+    def test__ensure_exists_default_true(self, project, add_gitlab_ci_config, capsys):
+        update_resource_group_config = f"""
+        projects_and_groups:
+          {project.path_with_namespace}:
+            resource_groups:
+              resource_group_that_dont_exist:
+                process_mode: newest_first
+            """
+
+        with pytest.raises(SystemExit) as exception:
+            run_gitlabform(update_resource_group_config, project)
+        assert exception.type == SystemExit
+        assert exception.value.code == EXIT_PROCESSING_ERROR
+        captured = capsys.readouterr()
+        assert "Project is not configured to use resource group" in captured.err
+
+    def test__ensure_exists_enforce_true(self, project, add_gitlab_ci_config, capsys):
```

Comment:
This test case and the next one both uses the term "enforce". Can we rename the test cases please?

You probably already know that some config of gitlabform supports a special key named "enforce". It's not implemented for resource group yet. In case it's added later, we probably have a naming clash. It might even be confusing now for others thinking about that enforce feature. That's why I think this term should be avoided in these tests.

What do you think?